test/e2e/manual/localnet.go

```diff
@@ -70,7 +70,7 @@ func main() {
 			log.Error(err)
 		}
 		log.Infoln("update config files")
-		for i := 0; i < 3; i++ {
+		for i := 0; i < len(tc.ValEthKs); i++ {
```

Comment:
Can we apply it in other places as well?

Reply:
also applied in multinode_setup. In other places, it's up to the test to decide how many nodes to use